docs/Community/Committers.md

```diff
@@ -0,0 +1,50 @@
+when to merge it. Remember that a PR must not be merged if a committer objects;
+instead, it should be brought up with the Community Governance Committee.
+
+| Name                  | GitHub Alias        | Area                           |
+|-----------------------|---------------------|--------------------------------|
+| Amaury Chamayou       | achamayou           | Build, CCF Integration         |
+| Anand Krishnamoorthi  | anakrish            | Debugging, SGX, EDL, Dev Tools |
+| Andrew Schwartzmeyer  | andschwa            | EDL, CMake, Dev Tools, Website |
+| Brett McLaren         | BRMcLaren           | Build, CMake, CI               |
+| Brian Telfer          | Britel              | TrustZone, Attestation         |
+| Simon Leet            | CodeMonkeyLeet      | SGX, APIs                      |
+| Dave Thaler           | dthaler             | TrustZone, APIs, Dev Tools     |
+| Emil Alexandru Stoica | EmilAlexandruStoica | Build, CMake, CI, Ansible      |
+| Akash Gupta           | gupta-ak            | SGX, Attestation, APIs         |
+| Hernan Gatta          | HernanGatta         | TrustZone                      |
+| Sergio Wong           | jazzybluesea        | Attestation, SGX               |
+| Jiri Appl             | jiria               | Attestation, TrustZone         |
+| John Kordich          | johnkord            | Build, CI, Dev Tools, Release  |
+| Jordan Hand           | jhand2              | Windows, Build, SGX            |
+| Xuejun Yang           | jxyang              | SGX                            |
+| Mike Brasher          | mikbras             | SGX, APIs, EDL                 |
+| Marius Oprin          | oprinmarius         | Build, CMake, CI, Ansible      |
+| Paul Allen            | paulcallen          | TrustZone                      |
+| Radhika Jandhyala     | radhikaj            | SGX, APIs, Website             |
+| Shruti Ratnam         | shruti25ratnam      | Attestation                    |
+| Cheng-mean Liu        | soccerGB            | Attestation, SGX               |
+| Bruce Campbell        | yakman2020          | Windows, SGX                   |
+
+Desired Status
+--------------
+
+As the project grows, in order to scale the management of the project, it is becoming increasingly necessary to delegate responsibility to domain-owners and improve the community's development workflow through automation. This will be handled through the use of [Prow](https://git.k8s.io/test-infra/prow).
+
+Commits should be fully gated through automated checks, with only a skeleton-crew of folks (a subset of the governance committee) able to directly merge code (and this power is only there for emergency measures, eg. if Prow has failed and needs to be restored).
+
+Ownership should be defined through directory-specific OWNERS files. All actions, such as initiating a test or merging code, should be performed through comments on GitHub that interact with the Prow bot.
```

Comment:
There are some problems we have with directory-specific ownership currently. Attestation code for example is not in a specific directory but is in mutliple directories interleaved with regular code.

Reply:
Fair enough. Some types of changes may always span across multiple SIGs.

Could the majority of attestation code lives in one location, and changes to other directories (other SIGs) would be more about interfaces, rather than implementation?

Reply:
This specific example is probably solved by the new attestation plugin design. For other examples I think we could better organize the code to avoid this muddiness.